e107_plugins/download/download.php

```diff
@@ -44,14 +44,15 @@
 
 	$dl = new download();
 
+	if(!defined("e_PAGETITLE")) {define("e_PAGETITLE", LAN_PLUGIN_DOWNLOAD_NAME);}
 
 	if(!defined("USER_WIDTH") && !deftrue('BOOTSTRAP')) { define("USER_WIDTH","width:100%"); }
 
 	/* define images */
 
 	if(deftrue('BOOTSTRAP'))
 	{
-		define("IMAGE_DOWNLOAD", (file_exists(THEME."images/download.png") ? THEME."images/download.png" : e_IMAGE."generic/download.png"));
+		define("IMAGE_DOWNLOAD", (file_exists(THEME."images/download.png") ? THEME."images/download.png" : 'icon-download.glyph'));
```

Comment:
It's a correction for a bootstrap site, and a missing icon bug for one that isn't.  ;-)